.jscsrc

```diff
@@ -27,7 +27,6 @@
         "allExcept": ["comments"]
     },
     "jsDoc": {
-        "checkTypes": "strictNativeCase",
```

Comment:
Why remove this?

Reply:
It's a duplicate object key and has two values (`strictNativeCase` and `true`).

https://github.com/Dash-Industry-Forum/dash.js/blob/46f33daae1cfd4599e9b440b60cef94a3b36bf6f/.jscsrc#L37